#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_size", int_t, 0, "Minimum size of bounding box", 1000, 100, 100000)
gen.add("max_size", int_t, 0, "Maximum size of bounding box", 1000000, 100, 100000)
exit(gen.generate(PACKAGE, "jsk_perception", "SelectiveSearch"))
